components/ProductManagement/Category/Category.tsx

```diff
@@ -1,26 +1,180 @@
+      accept: () => acceptRemove(id),
+      reject: rejectRemove,
+    });
+  };
+
+  const confirmAddCategory = async () => {
+    const newCategoryToSave = {
+      title: newCategory,
+      position: 1,
+    };
+    try {
+      const resultAddedCategory = await createCategory(newCategoryToSave, logout);
+      if (resultAddedCategory) {
+        showToast('info', 'Confirmed', 'The new category was added successfully.');
+        updateCategories();
+      }
+    } catch (error) {
+      console.error(error);
+    } finally {
+      setIsModalAddCategoryVisible(false);
+    }
+  };
+
+  const openAddNewCategoryModal = () => {
+    setNewCategory('');
+    setIsModalAddCategoryVisible(true);
+  };
 
-const Category: React.FC = () => {
   return (
     <div className='category-management'>
+      <Toast ref={toast} />
       <div className='category-management__mainbox'>
         <div className='category-management__mainbox__title'>
           <h2 className='poppins-600-regular'>Categories</h2>
         </div>
         <div className='category-management__mainbox__list'>
           <div className='category-management__mainbox__list__headers'>
             <div className='category-management__mainbox__list__headers__label'>
-              <h2 className='poppins-600-regular'>Category Title</h2>
+              <h2 className='poppins-600-regular'>Title</h2>
             </div>
             <div className='category-management__mainbox__list__headers__label'>
               <h2 className='poppins-600-regular'>Options</h2>
             </div>
           </div>
-          <div className='category-management__mainbox__list__rows'></div>
-          <div className='category-management__mainbox__list__actions'></div>
+          <div className='category-management__mainbox__list__rows'>
+            {categories?.map((category) => (
+              <RowCategory
+                key={category._id}
+                id={category._id}
+                label={category.title}
+                handleRemove={() => removeRow(category._id)}
+              />
+            ))}
+          </div>
+        </div>
+        <div className='category-management__mainbox__options'>
+          <Button
+            label='Add'
+            severity='info'
+            iconPos='left'
+            icon='pi pi-plus'
+            rounded
+            onClick={() => openAddNewCategoryModal()}
+          />
         </div>
       </div>
+      <UpdateModal
+        size='tiny'
+        open={isModalAddCategoryVisible}
+        dimmer='blurring'
+        closeModal={() => setIsModalAddCategoryVisible(false)}
+        handleCancel={() => setIsModalAddCategoryVisible(false)}
+        handleUpdate={confirmAddCategory}
+        header='Add Category'
+        cancelBtnLabel='Cancel'
+        updateBtnLabel='Add'
+      >
+        <section className='category-management__add-category'>
+          <span>Type the name for the new category and then press the `Add` button. </span>
```

Comment:
same here and the rest of the file, it has to be cleaned up from hardcoded strings